tests/format/flow/typeapp-call/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -27,18 +27,18 @@ f?.<T>(e);
 =====================================output=====================================
 //@flow
 f<T>();
-f < T > <U></U>;
+(f < T) > <U></U>;
```

Comment:
This is an issue, does someone have a suggestion on how to distinguish between this scenario and a sequence of comparison?